tools/docker/tests/test_flows.sh

```diff
@@ -631,6 +642,12 @@ main() {
         esac
     done
     [ -z "$*" ] && set ${ALL_TESTS}
+    if [ "$DUMP" = "true" ]; then
+	    tcpdump -i $(get_host_bridge_name) -U -w $DUMPFILE &
```

Comment:
it would be useful to comment the tcpdump options used in this command (nitpick)

Reply:
OK